lib/src/components/SimpleComponents.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import withClass from '../util/withClass';
+
+// Simple convenience components that are just a particular tag with a particular classname, with the ability
+// to add all attrs that you would normally be able to add to that tag (including more classnames)
+export const NxP = withClass('nx-p', 'p');
```

Comment:
One thing that I find odd about this is that the class is declared before the HTML tag, is that by design or required? I would expect to to look like: `export const NxP = withClass('p', 'nx-p');`

Reply:
Since both arguments are required I think it would be better to have the element name first.

Reply:
> Since both arguments are required I think it would be better to have the element name first.

e60d8101